.github/workflows/continuous-integration.yml

```diff
@@ -382,57 +385,29 @@ jobs:
           docker push "$image_name:latest"
 
 
-  # Upload built packages to https://build.rchain-dev.tk.
+  # GitHub (create) release and packages
   release_packages:
     name: Release Packages
     needs:
       - run_unit_tests
       - run_integration_tests
       - build_packages
-    if: "false && github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/staging')"
+    if: "github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')"
```

Comment:
I've added `v` prefix for tags that can trigger release. Also I removed _staging_ branch because release can only be created from a tag.